ic-os/components/ic/generate-ic-config/ic.json5.template

```diff
@@ -256,6 +256,8 @@ table ip6 filter {\n\
     icmpv6 type nd-router-advert accept\n\
     icmpv6 type nd-neighbor-solicit accept\n\
     icmpv6 type nd-neighbor-advert accept\n\
+    ip6 saddr { {{ ipv6_prefix }} } ct state { new } tcp dport { 9100, 19531, 19100 } accept\n\
+    ip6 saddr { {{ ipv6_prefix }} } udp dport { 9100, 19531, 19100 } accept\n\
```

Comment:
@dfinity/dre What ports should we open up for logs and metrics? Are TCP and UDP both needed?

Reply:
Can we also please allow 9090 and 9091? Only TCP is needed, we don't use UDP.